cli/src/telemetry/session.rs

```diff
@@ -53,6 +53,13 @@ pub struct Session {
 }
 
 impl Session {
+    /// This function will return an error if the api key is not defined
+    pub fn require_api_key(&self) -> Fallible<()> {
```

Comment:
Why not return `Fallible<String>` (or `&str` or whatever) ? It'd make the rest of the code simpler, no?